Clockwise.Redis/KeySapceObserver.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Threading.Tasks;
+using Pocket;
+using StackExchange.Redis;
+
+namespace Clockwise.Redis
+{
+    public sealed class KeySpaceObserver : IObservable<(string key, string operation)>, IDisposable
```

Comment:
Let's avoid `ValueTuple` in public interfaces.

Reply:
gone